README.md

```diff
@@ -290,6 +290,46 @@ If you try to read from a stream and pipe to a new file, you will obtain an empt
 
 ## JSON Schema body validation
 
+You can use `fastify-multipart` with schema validation, this is a full working example.
```

Comment:
Could you add a test for this new example?

We can make it consistent in this way